cmake/projects/drm/hunter.cmake

```diff
@@ -26,7 +26,7 @@ hunter_pick_scheme(DEFAULT drm)
 hunter_cacheable(drm)
 hunter_download(
     PACKAGE_NAME drm
-    PACKAGE_INTERNAL_DEPS_ID "1"
+    PACKAGE_INTERNAL_DEPS_ID "2"
```

Comment:
Actually I think we don't need this one here. Your patch doesn't fix broken packages, doesn't broke working packages, in general it doesn't affect how packages built - hence we don't need to change PACKAGE_INTERNAL_DEPS_ID.